packages/starters/backend/sirius-components-task-starter/src/main/java/org/eclipse/sirius/components/task/starter/configuration/TaskExampleBuilder.java

```diff
@@ -158,6 +173,114 @@ private Project createDailyProject(Person paul, Person peter) {
         return dailyProject;
     }
 
+    private Project createKanbanProject(Person paul, Person peter) {
+        Project kanbanProject = TaskFactory.eINSTANCE.createProject();
+        kanbanProject.setName("Kanban Project Dev");
+
+        List<TaskTag> kanbanTags = this.createKanbanTags();
+        kanbanProject.getOwnedTags().addAll(kanbanTags);
+
+        Task idea = TaskFactory.eINSTANCE.createTask();
+        idea.setName(IDEA);
+        idea.setStartTime(Instant.parse(DATE_2023_12_10T08_30_00Z));
+        idea.setEndTime(Instant.parse(DATE_2023_12_11T17_30_00Z));
+        idea.setProgress(50);
+        idea.getTags().add(kanbanTags.get(0));
+        Task spec = TaskFactory.eINSTANCE.createTask();
+        spec.setName(SPECIFICATION);
+        spec.setStartTime(Instant.parse(DATE_2023_12_11T08_30_00Z));
+        spec.setEndTime(Instant.parse(DATE_2023_12_12T17_30_00Z));
+        spec.setProgress(50);
+        spec.getDependencies().add(idea);
+        spec.getTags().add(kanbanTags.get(0));
```

Comment:
The _Idea_ and _Specification_ are in the To Do list but the _Release_ is done...